File: tuts/001-lightsail-gs/java/src/main/java/com/example/lightsail/GettingStartedScenario.java
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.lightsail;

import software.amazon.awssdk.services.lightsail.LightsailClient;

public class GettingStartedScenario {
    public static void main(String[] args) {
        try (LightsailClient client = LightsailClient.builder().build()) {
            LightsailWrapper wrapper = new LightsailWrapper(client);
            System.out.println("Running Lightsail getting started scenario...");
            // TODO: setup, interact, teardown
            System.out.println("Scenario complete.");
        }
    }
}

File: tuts/001-lightsail-gs/kotlin/src/main/kotlin/com/example/lightsail/GettingStartedScenario.kt
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.lightsail

import aws.sdk.kotlin.services.lightsail.LightsailClient
import kotlinx.coroutines.runBlocking

fun main() = runBlocking {
    LightsailClient { region = "us-east-1" }.use { client ->
        val wrapper = LightsailWrapper(client)
        println("Running Lightsail getting started scenario...")
        // TODO: setup, interact, teardown
    }
}
